adoc-mode.el

```diff
@@ -3493,7 +3757,11 @@ Turning on Adoc mode runs the normal hook `adoc-mode-hook'."
                    2 3 nil (1 . nil))))
   (when (boundp 'compilation-error-regexp-alist)
     (make-local-variable 'compilation-error-regexp-alist)
-    (add-to-list 'compilation-error-regexp-alist 'asciidoc)))
+    (add-to-list 'compilation-error-regexp-alist 'asciidoc))
+  (when adoc-show-images-at-startup
+    (adoc-display-images))
```

Comment:
I'm guessing you can wrap this in something like `ignore-errors` to avoid the CI issues.

Reply:
Thanks for pointing me there. I use now `(and (display-graphic-p) adoc-show-images-at-startup)` as condition. That does the trick.